test/test.js

```diff
@@ -1,11 +1,13 @@
-import fs from 'fs';
-import execa from 'execa';
-import test from 'ava';
-
-const exec = cmd => {
-	return execa.shell(cmd)
-		.catch(err => err)
-		.then(result => result.stdout);
+const fs = require('fs');
+const execa = require('execa');
+const test = require('ava');
+const path = require('path');
+
+const exec = async cmd => {
+	const {stdout} = await execa.shell(cmd)
+		.catch(error => error);
```

Comment:
IIRC there's an `execa` option for this.

Reply:
I can't find it, tbh. I haven't worked with `execa`  before, so maybe there is some option for that, but couldn't find it in its README. So would appreciate an advice on what to look at.

Reply:
`reject: false` but the `execa` version that was used here was quite old.